modules/shared/src/main/resources/logback-included.xml

```diff
@@ -157,6 +187,10 @@
         <appender-ref ref="GOSSIP" />
         <appender-ref ref="GOSSIP_JSON" />
     </logger>
+    <logger name="PeerSelectLogger" level="INFO" additivity="false">
```

Comment:
The code is logging a DEBUG message. I believe logging levels will write at the defined level and above, which means your debug message won't be written.

(Someone correct me if I'm mistaken...)

Reply:
The log level has been changed to `DEBUG`.